e2e/apprunner/apprunner_test.go

```diff
@@ -86,39 +86,6 @@ var _ = Describe("App Runner", func() {
 		})
 	})
 
-	Context("run storage init and svc deploy to create an S3 bucket and an aurora DB", func() {
-		var (
-			s3StorageInitErr error
-			auroraInitErr    error
-			deployErr        error
-		)
-
-		BeforeAll(func() {
-			_, s3StorageInitErr = cli.StorageInit(&client.StorageInitRequest{
-				StorageName:  "s3storage",
-				StorageType:  "S3",
-				WorkloadName: feSvcName,
-			})
-			_, auroraInitErr = cli.StorageInit(&client.StorageInitRequest{
```

Comment:
The `addon` test suite only test LBWS. I think s3 addon is probably fine, but there are something different with RDWS when it comes to RDS (iirc you need to have `network.placement: 'private'` to be able to send egress from RDWS to the DB). I think this was why we added RDS to `apprunner` test suite in the first place, and why we kept `copilot/front-end/manifest.yml` because we needed to keep that `network.placement: 'private'` bit.

Reply:
yeah - i figured that we were effectively testing the same thing (RDWS->VPC) because that's required for a RDWS service to reach a _private_ RDWS service. Traffic flows like: `front end -> VPC Connector -> VPC -> App Runner VPC Endpoint -> back-end`. So at least from a reachability standpoint, this change tests the same thing (it does stop testing RDWS+addons, but I think that's an ok tradeoff?)